container-build-checks.py

```diff
@@ -206,7 +206,7 @@ def check_image(image, result):
         if blocked_pattern is not None:
             result.warn(f"Tag {tag} is not allowed (blocked by {blocked_pattern}).")
 
-        if image.containerinfo["release"] in tag:
+        if image.containerinfo.get("release") in tag:
```

Comment:
FWICT this would run `if None in tag` which is a bit weird. What about:

```suggestion
        if "release" in image.containerinfo and image.containerinfo["release"] in tag:
```